distributed/cli/dask_ssh.py

```diff
@@ -29,12 +29,13 @@
               help="Port to use for SSH connections.")
 @click.option('--ssh-private-key', default=None, type=str,
               help="Private key file to use for SSH connections.")
-
+@click.option('--bindall', is_flag=True,
+              help="Let the worker bind to all available interfaces.")
```

Comment:
If I understand the code correctly, this is not true. If no host option is given, an interface which can reach the scheduler is automatically choosen. But if you provide a host option, `socket.gethostbyname` is used ([see here](https://github.com/dask/distributed/blob/master/distributed/cli/dask_worker.py#L142)).

This causes the real problem, if you specify a host like `foo` and your hosts file says that `127.0.0.1` is also known as `foo`, the worker will use `127.0.0.1` as the interface address.

After thinking about it a little bit, it might be better to rename the option to something like `--nohost` or `--bindauto`.